prov/gni/include/gnix_mr.h

```diff
@@ -176,23 +220,31 @@ typedef enum {
 } gnix_mrc_state_e;
 
 /**
+ * @brief  gnix memory registration cache entry storage
+ */
+struct gnix_mrce_storage {
+	atomic_t elements;
+	RbtHandle rb_tree;
+};
+
+/**
  * @brief  gnix memory registration cache object
  *
  * @var    state           state of the cache
  * @var    attr            cache attributes, @see gnix_mr_cache_attr_t
- * @var    inuse           red-black tree containing in-use memory registrations
- * @var    stale           reb-black tree containing stale memory registrations
- * @var    inuse_elements  count of in-use memory registrations
- * @var    stale_elements  count of stale memory registrations
+ * @var    inuse           cache entry storage struct
+ * @var    stale           cache entry storage struct
+ * @var    lru_head        head of LRU eviction list
+ * @var    rbtlist_free    freelist of cache entries for fast allocation
```

Comment:
This doesn't match up with what's actually in the struct.  There is no `rbtlist_free`, but there are `hit` and `misses`.